### ISSUE TITLE
Not able to drag and resize

### ISSUE THREAD

**Issue by chinchang** [nathancahill/Split.js#141]
I am using the save state example from readme to save the sizes in localstorage and apply on next refresh. 
Example of a saved state: `['calc(50% - 2px)', 'calc(50% - 2px)']`. 
The problem is, in the latest version of the library, [in `adjust` function](https://github.com/nathancahill/Split.js/blob/9efba63c9de6d1dd7d5ce75b791576afc5125f33/src/split.js#L191), `percentage` is calculated as sum of `a.size` and `b.size`. But the sizes are actually strings. So finally my `percentage` variable has `'calc(50% - 2px) calc(50% - 2px)'`. And hence dragging calculations fail.

@nathancahill Is that a bug or something wrong on my end?

**Comment by chinchang** [comment]
Hi Nathan

Here is the demo of the issue -> https://codepen.io/chinchang/pen/xyymOw
If you click on 'Set in numbers' and then drag gutters, it works.
But if you click on 'Set in calc' and then drag, it doesn't work. According to docs, I think passing `calc` is allowed...or not?

**Comment by nathancahill** [comment]
No, it looks like the docs are wrong in this regard. What you would want to do in this case is get the total width, and calculate the percentages based on that: https://codepen.io/anon/pen/NOEPPY

**Comment by chinchang** [comment]
Cool. Thanks.
